install-requirements.sh

```diff
@@ -2,7 +2,15 @@
 
 set -e
 
+# install dependencies
 for file in `ls functions/*/requirements.txt`
 do
   pip install --upgrade -r $file -t `dirname $file` 
 done
+
+# copy shared tools
+pip install --upgrade -r tools/requirements.txt -t tools
+for dir in `ls -d functions/*/`
+do
+  cp -R tools $dir
```

Comment:
ok I added this except for the clean hook. I tried putting everything in a sub dir unsuccessfully. After looking at the docs more it looks like lambda only supports storing dependencies at the root.